app/auth/routes.py

```diff
@@ -155,23 +156,25 @@ def register():
             message="Email and password must included in the request body"
         )
 
-    full_name = r.get("fullname", None)
+    first_name = r.get("firstName", None)
+    last_name = r.get("lastName", None)
     email = r.get("email", None)
     password = r.get("password", None)
-    session_id = r.get("sessionId", None)
+    quiz_uuid = r.get("quizId", None)
 
-    if not valid_name(full_name):
+    if not valid_name(first_name) or not valid_name(last_name):
```

Comment:
I think the errors should be separate for the first and last name, unless the front end is also requiring form validation so the user knows their first name is the problem or their last name is the problem. If the user is just told either their first name or their last name (but not which one exactly) is causing problems, that is likely to frustrate them.